source/scripting_v3/GTA/Entities/Peds/Ped.cs

```diff
@@ -1124,5 +1125,10 @@ public string MovementAnimationSet
 		}
 
 		#endregion
+
+		public static PedHash[] GetAllPedModels()
```

Comment:
This is inconsistent with `Vehicle.GetAllModels()`. I think I'd be better to call these `Ped.GetAllModels()` and `Weapon.GetAllModels()` or `Weapon.GetAllObjectModels()`.